pkgs/jni/test/jobject_test.dart

```diff
@@ -175,6 +175,8 @@ void run({required TestRunnerCallback testRunner}) {
       });
     });
     expect(randomInt, lessThan(15));
+    final JObject? nullableJObject = null;
+    expect(nullableJObject.use(() => 'foo'),equals('foo));
```

Comment:
This test has syntax errors! The callback should actually get an argument and also a single quote is missing.

Reply:
I guess editing on GitHub is not the way to go. 
I'm really sorry